src/test/java/io/vertx/test/core/LocalEventBusTest.java

```diff
@@ -1128,69 +1121,6 @@ public void testUnregisterThenUnsetEndHandler() {
     await();
   }
 
-  private void testUnregistrationOfUnregisteredConsumerCallsEndHandler(MessageConsumer<String> consumer, ReadStream<?> readStream) {
```

Comment:
they were actually testing the previous behavior, one test was kept to check endHandler is not called when unregistering and only when eventbus is closing.